quick_purchase/models/purchase_order.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import ast
+from openerp import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def add_product(self):
+        self.ensure_one()
+        action = self.env.ref('quick_purchase.product_product_action')
+        context = ast.literal_eval(action.context or "{}").copy()
+        context.update({'purchase_id': self.id})
+        result = action.read()[0]
+        result.update(
+            {'target': 'current', 'context': context, 'view_mode': 'tree'})
+        return result
```

Comment:
Action name is 'Purchase Products'.
Purchase is displayed in breadcrumb. Only Seller name is missing in breadcrumb.
May be we can complete name key here 'Purchase Products (my supplier)'